docs/source/blocks/block-style-wrapper.md

```diff
@@ -0,0 +1,99 @@
+---
+html_meta:
+  "description": "The block style wrapper part of a block anatomy, allows you to inject styles from the block schema into the block wrapper in form of class names."
+  "property=og:description": "The block style wrapper part of a block anatomy, allows you to inject styles from the block schema into the block wrapper in form of class names."
+  "property=og:title": "Blocks - Style Wrapper"
+  "keywords": "Volto, Plone, frontend, React, Blocks, Edit, Style, Wrapper, components"
+---
+
+# Blocks - Style Wrapper
+
+The block style wrapper is part of a block anatomy.
+It allows you to inject styles from the block schema into the block wrapper in the form of class names.
+It wraps the block edit and the view components.
+
+It can be used mapping directly unique values to CSS properties, like `background-color` mapped to a single color.
```

Comment:
```suggestion
It can be used by directly mapping unique values to CSS properties.
For example, `background-color` could be mapped to a single color.
```